tests/abi/abi.c

```diff
@@ -0,0 +1,175 @@
+/*
+ * This file is distributed under the MIT License. See LICENSE.md for details.
+ */
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "for_each.h"
+#include "functions.h"
+
+#ifdef __x86_64__
+#include "platform_specific/x86_64.h"
+#elif __x86__
+#include "platform_specific/x86.h"
+#else
+_Static_assert(0, "unsupported platform architecture");
+#endif
+
+/* LFSR generator */
+
+static uint64_t LFSR = 0xdc984ef2647af904ull;
```

Comment:
For C files we usually use snake_case, all lower case, see other C files in revng-qa.

Reply:
Done.